.github/workflows/ci.yml

```diff
@@ -8,8 +8,8 @@ jobs:
     strategy:
       matrix:
         ruby:
-          - 2.6
```

Comment:
We need to change the gemspec to require Ruby >= 2.7, but Ruby 2.6 is still supported at Shopify so I think we should continue to support here.

Reply:
That would mean that I need to test against two different Rails versions, as Rails 7 doesn't support Ruby 2.6.

Reply:
Fixed this by setting up a proper build matrix and multiple Gemfiles.